embrace-android-sdk/src/main/cpp/unwinders/stack_unwinder.cpp

```diff
@@ -0,0 +1,70 @@
+//
+// Created by Eric Lanz on 5/12/20.
+//
+
+#include "stack_unwinder.h"
```

Comment:
i assume this file was just moved and not changed, so i didn't review it. LMK if that was a bad assumption

Reply:
That is correct, the only change I made was to rename a function